randovania/data/json_data/prime3.txt

```diff
@@ -3684,15 +4527,18 @@ Leads to Command Center/Dock to Metroid Creche
       Trivial
 
 > Event - Metroid Hatcher; Heals? False
-  > Pickup (Energy Tank)
-      Spider Ball and Boost Ball
+  * Event Metroid Hatcher (Pirate Homeworld)
+  > Bottom Floor
+      Trivial
 
 > Bottom Floor; Heals? False
   > Dock to Creche Transit
       Trivial
   > Dock to Transit Station 3-A
 Leads to Command Center
       Use Screw Attack (Space Jump)
+  > Pickup (Energy Tank)
+      Spider Ball and Boost Ball and After Metroid Hatcher (Pirate Homeworld)
   > Event - Metroid Hatcher
       All of the following:
           Boost Ball
```

Comment:
For Transit Station 3-A (Below):
- Add node "Tram to Command Center - Transit Station 3-B" and connect to save station door as Trivial. Make this a teleporter node.
- Connect save station door to tram as trivial